HomePage.html

```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
 						</ul>
 
 					<button class="scroll-button">
-						<img src="Assets/Icons/scroll-button.svg" alt=""/>
+						<a href="#projects"><img src="Assets/Icons/scroll-button.svg" alt=""/></a>
 					</button>
 		
 				</section>
 	
+		</div>
+
+		
+		<div class="project-page" id="projects">
+			
+			<section >
+				<h1>My Recent <br> Works</h1>
```

Comment:
[REQUIRED] Kindly use another heading tag You should have only one `h1` tag in your document. This is more accessible and helpful for screen readers.